project-management/Github.md

```diff
@@ -0,0 +1,77 @@
+# Managing public (or private) projects using GitHub
+
+We use a common set of labels across projects in GitHub. It is
+possible to precisely control this set of labels using
+the [settings GitHub app][github-app-settings] (needs to be enabled
+for your org). Use the [`settings.yml`](../.github/settings.yml) from
+this repository as a starting point.
+
+[github-app-settings]: https://github.com/probot/settings
+
+## Labels
+
+We use labels to classify tasks by priority, from P0 to P4:
+
+* **P0 (blocker)** Do not do anything else, this is a blocker.
+* **P1 (critical)** Must have.
+* **P2 (major)** Should have.
+* **P3 (minor)** Could have.
+* **P4 (trivial)** Someday/maybe. Consider closing as "wontfix".
+
+Labels are also useful to classify issues by type or by component. Use
+colours to distinguish whether a label designates a type, a component
+or something else.
+
+## Project leads
+
+Each project must have a project lead (maintainer). By convention, the
+PL is whoever appears at the top of [`CODEOWNERS`][github-codeowners]
+file, e.g.:
+
+```
+* @mboes
+```
+
+This has the effect that @mboes will automatically be tagged as
+a reviewer for all incoming PR's, unless later lines in the same file
+override that.
+
+[github-codeowners]: https://help.github.com/articles/about-codeowners/
+
+## Milestones
+
+We gate tickets with milestones. Milestones always have an associated
+due date.
+
+## Releases
+
+Every public release must include a `CHANGELOG.md` file, in the format
+of [keepachangelog]. New entries to the changelog are integral parts
+of bug fix and feature PR's.
+
+Each release has an associated tag of the form `v$VERSION`, GPG-signed
+by the project lead (use `git tag -s`).
+
+[keepachangelog]: https://keepachangelog.com/en/1.0.0/
+
+## Documentation
+
+Every document has a *document owner*. The owner is usually specified
+in the document itself. By default, the project lead is the owner for
+all documentation in the project.
+
+We do not use [wikis][github-wiki] for documentation. The reasons for
+this are well explained [elsewhere][yesod-documentation-thoughts].
+When using [GitHub pages][github-pages], we *do not* use a separate
+`gh-pages` branch to hold documentation. The reason is that this
+breaks the ability to update documentation atomically with the
+associated code changes. We typically keep the documentation in
+a `docs/` folder. This can readily be deployed to a public site
+using [GitHub pages][github-pages-conf]
+or [Read the Docs][readthedocs].
```

Comment:
I think we should have a default policy here, for open source projects. Should we use github pages? or should we use read the docs? Or both?

I don't really care, but there should be a default policy for the company, this will take one more thing out of the mind of developers.